recipes/spectral/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "spectral" %}
+{% set version = "0.18" %}
+{% set sha256 = "eb96ad3ba8f2fcf5c3cb0ca1fab92deafe1ae806bcba28d3e187ee15033c87d1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - spectral
+
+about:
+  home: https://github.com/spectralpython/spectral/
+  license: GPLv2
+  license_family: GPL
+  license_file: LICENSE.txt
+  summary: 'A Python module for hyperspectral image processing.'
+  description: |
+    A pure Python module for processing hyperspectral image data (imaging spectroscopy data). 
+    It has functions for reading, displaying, manipulating, and classifying hyperspectral 
+    imagery. SPy is Free, Open Source Software (FOSS) distributed under the GNU General Public License.
+  doc_url: http://spectralpython.net
+  dev_url: https://github.com/spectralpython/spectral/
+
+extra:
+  recipe-maintainers:
+    - lewismc
+    - tboggs
```

Comment:
@tboggs this concerns adding spectral to the conda-forge platform and making it available through the conda channel. I had to create a recipe for spectral as we consume it within pycoal. Thanks in advance, if you are NOT OK with being an official maintainer then I can remove you from this section. Thank you in advance.